help/include/lists.md

```diff
@@ -0,0 +1,33 @@
+## Bulleted list syntax
+
```

Comment:
I think "syntax" sounds too technical; let's just make it "Bulleted lists" and "Numbered lists".